app/components/nav-bar/template.hbs

```diff
@@ -0,0 +1,14 @@
+<!-- Navbar -->
+    <nav class="navbar navbar-inverse">
```

Comment:
It is up to you guys. Personally, I prefer darker but to be consistent with the design we may have to change.